test/powershell/Modules/Microsoft.PowerShell.Management/FileSystem.Tests.ps1

```diff
@@ -529,6 +529,11 @@ Describe "Handling of globbing patterns" -Tags "CI" {
             Test-Path -LiteralPath $testPath2 | Should -BeTrue
         }
     }
+
+    It "Handle wildcards in root of device path" -Skip:(!$IsWindows) {
```

Comment:
You should add a test case for `\\?\` path. you should be able to get one using:

```powershell
(get-ciminstance win32_volume | select-object -first 1).deviceid
```

Reply:
It seems to return null in the test environment. On my own PC it returns a volume, even without running as admin so I don't know if it's related to permissions. Any ideas? Can I just change it to: `'\\`?\C:\*'`?

Reply:
Thanks for checking.  I think we will have to skip this test then if the CI environment doesn't support it.

Reply:
👍 I've updated the test to be identical to the other, except with a `?` instead so we also test if escaping it works properly.